src/components/ContentHighlights/ZeroStateCardText.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Card } from '@edx/paragon';
+import PropTypes from 'prop-types';
+
+const ZeroStateCardText = ({ textContainerClassName, children }) => (
+  <Card.Section className={textContainerClassName}>
+    {children}
+  </Card.Section>
+);
+
+ZeroStateCardText.propTypes = {
+  textContainerClassName: PropTypes.string,
+  children: PropTypes.node,
```

Comment:
nit: Should `children` be required here? It doesn't really make sense to render this component without `children` :)